src/socket.ts

```diff
@@ -0,0 +1,54 @@
+import type { Socket } from 'socket.io-client'
```

Comment:
Not sure if we want to use the `webrtc-peer` library. It should be simpler to just use the RTCPeerConnection.